### PR TITLE
Automated testing of organizational settings

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
Automated testing of organizational settings

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/auto_test/tests/org/organization-settings.spec.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  title,
+  modifyTitle,
+  memberName: '吴辉洛',
+  manager: '组织管理员',
+  image: 'app/images/Erda.png',
+  memberBoss: 'erda-ui-team',
+};
+
+Role('Manager', () => {
+  test.only('organization-settings', async ({ wait, page, expectExist, goTo }) => {
```
Comment (Zero-Rock) [inline shell/auto_test/tests/org/organization-settings.spec.ts:28]: remove only

File: shell/auto_test/tests/org/organization-settings.spec.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    const base = new Base(page);
+    await goTo('organizationMember');
+    await page.click('button:has-text("add member")');
+    await page.click('.ant-select.w-full .ant-select-selector .ant-select-selection-overflow');
+    await page.fill('text=member Please enter a member name to search >> input[role="combobox"]', testData.memberName);
+    await page.click('.ant-select-item-option-content');
+    await page.click(
+      '.ant-form div:nth-child(3) .ant-col.ant-form-item-control .ant-form-item-control-input .ant-form-item-control-input-content .ant-select .ant-select-selector .ant-select-selection-overflow',
```
Comment (Zero-Rock) [inline shell/auto_test/tests/org/organization-settings.spec.ts:36]: add comments for complex selectors

File: shell/auto_test/tests/org/organization-settings.spec.ts (hunk 1 above)
Comment (Zero-Rock) [reply shell/auto_test/tests/org/organization-settings.spec.ts:28]: follow the menu to split the test suit

File: shell/auto_test/tests/org/organization-settings.spec.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    await page.fill('#displayName', testData.title);
+    await page.click('span:has-text("Chinese")');
+    await page.click('text=English');
+    await page.click('text=English');
+    await page.click('div[role="document"] >> text=Chinese');
+    await page.click('text=private org');
+    await page.click('span:has-text("public org")');
+    await page.click(
+      'svg:has-text("@font-face{font-family:feedback-iconfont;src:url(//at.alicdn.com/t/font_1031158_")',
+    );
```
Comment (Zero-Rock) [inline shell/auto_test/tests/org/organization-settings.spec.ts:132]: url is mutable，don‘t use unstable selector

File: shell/auto_test/tests/org/organization-settings.spec.ts [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    await page.click(
+      'span[role="button"]:has-text("@font-face{font-family:feedback-iconfont;src:url(//at.alicdn.com/t/font_1031158_")',
+    );
```
Comment (Zero-Rock) [inline shell/auto_test/tests/org/organization-settings.spec.ts:135]: same as above

File: shell/auto_test/tests/org/organization-settings.spec.ts (hunk 1 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-settings.spec.ts]: ok,modified

File: shell/auto_test/tests/org/organization-settings.spec.ts (hunk 3 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-settings.spec.ts]: ok,modified

File: shell/auto_test/tests/org/organization-settings.spec.ts (hunk 4 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-settings.spec.ts]: ok,modified

File: shell/auto_test/tests/org/organization-settings.spec.ts (hunk 2 above)
Comment (wuhuiluo) [reply shell/auto_test/tests/org/organization-settings.spec.ts]: ok,modified

**Pull request merged**